ocp_resources/cluster_role.py

```diff
@@ -4,8 +4,7 @@
 
 class ClusterRole(Resource):
     """
-    ClusterRole in kubernetes 'authorization-resources' official API:
-        https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/
+    https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/
```

Comment:
What was fixed here? this is the same link no?

Reply:
In this discussion I was asked to remove the text above mentioning the API
https://github.com/RedHatQE/openshift-python-wrapper/pull/1353#discussion_r1301186544

Reply:
so please fix the pr title